localstack/utils/aws/templating.py

```diff
@@ -128,7 +128,21 @@ def calculate(fn, self, *args, **kwarg):
     return result
 
 
+class ExtAssignment(airspeed.Assignment):
+    """
+    Extends the airspeed Assignment class to support names with dashes, e.g., "X-Amz-Target"
+    """
+
+    START = re.compile(
```

Comment:
I personally think it would be better if we maintain these extensions in the airspeed repo. As we've discussed previously, we need to anyway fix/refactor some of the tests in airspeed (as they seem to contain some incorrect assumptions that do not conform to the VTL spec and/or the behavior in AWS).

Fine with merging this as-is here for now, but let's try to port these changes into airspeed and enhance the test parity over there in a follow-up soon. 👍

Reply:
Yep, totally - and just to be clear, [our fork](https://github.com/localstack/airspeed) has already diverged significantly from upstream `airspeed`, so I think we can just continue to keep iterating on the functionality in our fork (focusing on VTL/AWS parity), no need to reconcile with upstream at this point. 👍